include/tallies/tallies.hpp

```diff
@@ -64,6 +71,33 @@ class Tallies {
 
   static Tallies& instance();
 
+  //===============================================
+  // addding filters instances into the Map
+  void add_position_filter_set(std::size_t id,
+                               std::shared_ptr<PositionFilter> filter);
+  void add_cartesian_filter_set(std::size_t id,
+                                std::shared_ptr<CartesianFilter> filter);
+  void add_cylinder_filter_set(std::size_t id,
+                               std::shared_ptr<CylinderPositionFilter> filter);
+  void add_energy_filter_set(std::size_t id,
+                             std::shared_ptr<EnergyFilter> filter);
```

Comment:
```suggestion
  void add_position_filter(std::size_t id,
                               std::shared_ptr<PositionFilter> filter);
  void add_cartesian_filter(std::size_t id,
                                std::shared_ptr<CartesianFilter> filter);
  void add_cylinder_filter(std::size_t id,
                               std::shared_ptr<CylinderPositionFilter> filter);
  void add_energy_filter(std::size_t id,
                             std::shared_ptr<EnergyFilter> filter);
```
These are going to be great ! Let's just shorten the name a bit is all.